divinediscipline/send-it
# #162894416 Add more integration/unit tests

**What does this PR do?**
Add more integration/unit tests

**Description of task to be completed**
- write more integration/unit tests for api endpoints

**How should this be manually tested?**
Clone the project, run 'npm install' to install all dependencies, create database, check db.js on the root folder and the connection script in the models folder, input the appropriate database and admin details for the environmental variables with appropriate values. Run 'npm test' to create tables, start the server and run tests.

**Any background context you want to provide?**
None

**What are the relevant pivotal tracker stories?**
#162894416
